src/libraries/Microsoft.PowerFx.Connectors/Execution/HttpFunctionInvoker.cs

```diff
@@ -63,31 +65,56 @@ public HttpRequestMessage BuildRequest(FormulaValue[] args)
             // Header names are not case sensitive.
             // From RFC 2616 - "Hypertext Transfer Protocol -- HTTP/1.1", Section 4.2, "Message Headers"
             var headers = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);
+            string body = null;
 
             var map = _argMapper.ConvertToSwagger(args);
-            foreach (var param in _argMapper._openApiParameters)
+            foreach (var param in _argMapper.OpenApiParameters)
             {
-                if (map.TryGetValue(param.Name, out var paramValue))
+                if (param.In == FxParameterLocation.Body && param.Schema.Properties != null && param.Schema.Properties.Any())
+                {
+                    var props = new Dictionary<string, object>();
+
+                    foreach (var property in param.Schema.Properties)
+                    {
+                        if (map.TryGetValue(property.Key, out var paramValue))
+                        {
+                            props.Add(property.Key, paramValue.ToObject());
+                        }
+                    }
+
+                    body = _argMapper.ContentType switch
+                    {
+                       "application/x-www-form-urlencoded" => props.ToFormUrlEncoded(),
```

Comment:
>application/x-www-form-urlencoded" [](http://example.com/codeflow?start=24&length=34)

we should make these constants somewhere
(see OpenApiExtensions._applicationJson) 

ARe these case sensntive?

Reply:
actually ... since this is a specific list we need to implement (and not just a string we pass through), should this be an enum?